src/arbcall.jl

```diff
@@ -99,11 +103,11 @@ end
 
 function jlfname(
     arbfname,
-    prefixes = ("arf", "arb", "acb", "mag", "mat"),
-    suffixes = ("si", "ui", "d", "arf", "arb");
+    prefixes = ("arf", "arb", "acb", "mag", "mat", "vec"),
+    suffixes = ("si", "ui", "d", "mag", "arf", "arb", "mpfr", "str");
     inplace = false,
 )
-    strs = split(arbfname, "_")
+    strs = filter(!isempty, split(arbfname, "_"))
```

Comment:
I like the idea of keeping the first underscore in the parsed name. In Arb they mark more unsafe functions for which you have to be careful with the arguments. Julia also uses it in a similar fashion for internal methods. So keeping the underscore would make them stand out more. What do you think?